tooling/maven/package-maven-plugin/src/main/java/org/apache/camel/quarkus/maven/PrepareCatalogQuarkusMojo.java

```diff
@@ -63,7 +63,7 @@
     private static final String DEFAULT_FIRST_VERSION = "0.2.0";
 
     private static final String[] EXCLUDE_EXTENSIONS = {
-            "http-common", "jetty-common", "support", "xml-common", "xstream-common"
+        "core-xml", "http-common", "jetty-common", "support", "xstream-common"
```

Comment:
Should `core-xml` really be here, given we said it is end-user facing? 

Besides, we currently seem to exclude all `-common` extensions, so the list could perhaps be simplified to `["-common", "support"]`